integration-cli/docker_api_exec_resize_test.go

```diff
@@ -19,8 +20,8 @@ func (s *DockerSuite) TestExecResizeApiHeightWidthNoInt(c *check.C) {
 
 	endpoint := "/exec/" + cleanedContainerID + "/resize?h=foo&w=bar"
 	status, _, err := sockRequest("POST", endpoint, nil)
-	c.Assert(status, check.Equals, http.StatusInternalServerError)
-	c.Assert(err, check.IsNil)
+	c.Assert(status, checker.Equals, http.StatusInternalServerError)
+	c.Assert(err, checker.IsNil)
```

Comment:
It should be better to move this error assertion before the HTTP status code assertion because if error occured the status might had not been initialized.